projects/fsastorefrontstyles/scss/overrides/category-page.scss

```diff
@@ -96,6 +102,32 @@ cx-page-layout.FSCategoryPageTemplate {
         }
       }
     }
+    cx-fs-cms-custom-container.categoryInnerContainer {
```

Comment:
- We don't use camel case for styling classes. Please use dash.
- Since you can pass multiple classes, perhaps all this css is not needed then.